html/Mogo/index.html

```diff
@@ -701,6 +701,14 @@ <h2 class="section-title section-title--black projects__title">
 
     <!-- Gallery -->
     <section class="gallery">
+
+      <div class="gallery__show">
```

Comment:
gallery__popup/gallery__modal but show not very good name here or gallery-popup/gallery-popup-wrapper